src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/AsyncMethodBuilderCore.cs

```diff
@@ -92,6 +93,8 @@ internal static bool TrackAsyncMethodCompletion
         /// <summary>Gets a description of the state of the state machine object, suitable for debug purposes.</summary>
         /// <param name="stateMachine">The state machine object.</param>
         /// <returns>A description of the state machine.</returns>
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2006:UnrecognizedReflectionPattern",
```

Comment:
Can we close [https://github.com/dotnet/runtime/issues/32921](https://github.com/dotnet/runtime/issues/32921) with this?

Reply:
Yup